packages/fxa-settings/src/lib/auth-errors/auth-errors.test.ts

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { composeAuthUiErrorTranslationID, AuthUiErrorNos } from './auth-errors';
+import * as Sentry from '@sentry/browser';
+
+const notAnExistingErrorNumber = 100000;
+const errorWithNoErrorNumber = {
+  message: "I'm an error with no error number!",
+};
+const errorWithAnInvalidErrorNumber = {
+  errno: notAnExistingErrorNumber,
+  message: 'This is an error with an invalid errno!',
+};
+
+jest.mock('@sentry/browser', () => ({
+  captureMessage: jest.fn(),
+}));
+
+describe('composeAuthUIErrorTranslationID', () => {
```

Comment:
Heh I was going to say you _could_ update your function name to match the casing in the `AuthUiError` type but I think the `describe` casing is off instead.

On that note though I think we use `Id` in the codebase more than `ID` or at least, I see where we refer to `ftlId`s. If you wanted this to be slightly shorter too you could call it `composeAuthUiErrorFtlId` or even `getAuthUiErrorFtlId` but also eh I don't really care too much or I would have called it out earlier, I'm just nitpicking at this point since I'm r+'ing. 😂